apps/framework-cli/src/framework/python/scripts_worker.rs

```diff
@@ -74,14 +74,13 @@ pub async fn start_worker(project: &Project) -> Result<Child, WorkerProcessError
         while let Ok(Some(line)) = stdout_reader.next_line().await {
             let parts: Vec<&str> = line.split('|').collect();
             if parts.len() >= 3 {
-                let level = parts[0];
-                let name = parts[1];
```

Comment:
The name was always 'python_worker_wrapper.logging' - not sure how useful that is

Reply:
Good call